ws/src/server.rs

```diff
@@ -64,6 +64,8 @@ impl Server {
 			config.max_connections = max_connections;
 			// don't accept super large requests
 			config.max_fragment_size = max_payload_bytes;
+			config.max_in_buffer_capacity = max_payload_bytes;
```

Comment:
Initially I thought that maybe it's too little, but the default 5MB for this should be more than enough for a bunch of requests.